javaparser-core/src/main/java/com/github/javaparser/ast/visitor/DumpVisitor.java

```diff
@@ -132,7 +132,7 @@ public void printLn(final String arg) {
 		}
 
 		public void printLn() {
-			buf.append("\n");
+			buf.append(System.getProperty("line.separator"));
```

Comment:
Looks good, Java 7 has System.lineSeparator but I think it is the best we can do with our JDK 6 constraint